defender/downloaded/Contract_Sentinel_cToken_Discord.js

```diff
@@ -0,0 +1,346 @@
+/* eslint-disable import/no-extraneous-dependencies,import/no-unresolved */
+const axios = require('axios');
+const ethers = require('ethers');
+
+// import the DefenderRelayProvider to interact with its JSON-RPC endpoint
+const { DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
+/* eslint-enable import/no-extraneous-dependencies,import/no-unresolved */
+
+const TOKEN_ABI = [
+  'function decimals() view returns (uint8)',
+  'function symbol() view returns (string)',
+];
+const MAKER_TOKEN_ABI = [
+  'function decimals() view returns (uint256)',
+  'function symbol() view returns (bytes32)',
+];
+const CTOKEN_ABI = ['function underlying() view returns (address)'];
+
+const makerTokenAddress = '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'.toLowerCase();
+const saiTokenAddress = '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'.toLowerCase();
+const oddTokens = [makerTokenAddress, saiTokenAddress];
+const cEtherAddress = '0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5'.toLowerCase();
```

Comment:
It seems unnecessary to lowercase these addresses, and as a result the subsequent logic in this file also needs to lowercase any address it comes across.

Reply:
Fixed.